docs/modules/ROOT/pages/clc-viridian.adoc

```diff
@@ -296,11 +293,49 @@ Parameters:
 
 |===
 
+== clc viridian stream-logs
+
+Outputs the logs of the given Viridian cluster as a stream.
+
+Make sure you authenticate to the Viridian API using `viridian login` before running this command.
+
+The log format may be one of:
+
+* minimal: Only the log message
+* basic: Time, level and the log message
+* detailed: Time, level, thread, logger and the log message
+* free form template, see: https://pkg.go.dev/text/template for the format.
+You can use the following placeholders: `msg`, `level`, `time`, `thread` and `logger`.
+Usage:
+
+[source,bash]
+----
+clc viridian stream-logs [cluster-ID/name] [flags]
+----
+
+Parameters:
+
+[cols="1m,1a,2a,1a"]
+|===
+|Parameter|Required|Description|Default
+
+|`--api-key`
+|Optional
+|Sets the API key. Overrides the `CLC_VIRIDIAN_API_KEY` environment variable. If not given, one of the existing API keys will be used.
+|
```

Comment:
Can we also add `cluster-ID/name` as a parameter?

Reply:
It seems we didn't add that to other `viridian` commands.